### ISSUE TITLE
inconsistency between upper and lower target

### ISSUE THREAD

**Issue by dschwoerer** [bendudson/hermes-3#216]
On the lower target `sin_alpha` is missing:
https://github.com/bendudson/hermes-3/blob/cf5d2967622eb841e5c2245995f872b7fbece308/src/sheath_boundary.cxx#L250

![Screenshot at 2024-03-25 10-20-59](https://github.com/bendudson/hermes-3/assets/5637662/cdc5ddbe-feca-4e72-aee8-16242a426673)

**Comment by dschwoerer** [comment]
There are also two different expressions for `s_i` used:
```c++
        BoutReal s_i = (nesheath > 1e-5) ? nisheath / nesheath : 0.0; // Concentration                                                                                                                                                       
        BoutReal s_i = clip(nisheath / floor(nesheath, 1e-10), 0, 1); // Concentration 
```

I am currently writing a replacement for the sheath_boundary component, that will not only unify upper and lower target, but also FCI and non-FCI ...

**Comment by bendudson** [comment]
This was fixed in 2faaa3c5
